R/stats_wrapper.R

```diff
@@ -228,6 +228,12 @@ do_cmdscale_ <- function(df,
 
   # this is executed on each group
   do_cmdscale_each <- function(df){
+    if (all(df[[value_col]]==0)) {
```

Comment:
What about other corner cases such as all NA, all Inf/-Inf case? Are those already taken care of? or it shouldn't happen at there?

Reply:
Thanks for checking out.